week2/week2-NER.ipynb

```diff
@@ -194,7 +194,7 @@
     "    \n",
     "    # Mapping tok2idx should contain each token or tag only once. \n",
     "    # To do so, you should:\n",
-    "    # 1. extract unique tokens/tags from the tokens_or_tags variable, which is not\n",
+    "    # 1. extract unique tokens/tags from the tokens_or_tags variable, which do not\n",
     "    #    occure in special_tokens (because they could have non-empty intersection)\n",
```

Comment:
occur

Reply:
You're right @rayryeng, thank you, missed that